examples/api/lib/material/navigation_rail/navigation_rail.0.dart

```diff
@@ -92,88 +92,74 @@ class _NavRailExampleState extends State<NavRailExample> {
                 const SizedBox(height: 20),
                 Text('Label type: ${labelType.name}'),
                 const SizedBox(height: 10),
-                OverflowBar(
-                  spacing: 10.0,
-                  children: <Widget>[
-                    ElevatedButton(
-                      onPressed: () {
-                        setState(() {
-                          labelType = NavigationRailLabelType.none;
-                        });
-                      },
-                      child: const Text('None'),
-                    ),
-                    ElevatedButton(
-                      onPressed: () {
-                        setState(() {
-                          labelType = NavigationRailLabelType.selected;
-                        });
-                      },
-                      child: const Text('Selected'),
-                    ),
-                    ElevatedButton(
-                      onPressed: () {
-                        setState(() {
-                          labelType = NavigationRailLabelType.all;
-                        });
-                      },
-                      child: const Text('All'),
-                    ),
-                  ],
-                ),
+                SegmentedButton<NavigationRailLabelType>(
+                    segments: const <ButtonSegment<NavigationRailLabelType>>[
+                      ButtonSegment<NavigationRailLabelType>(
+                        value: NavigationRailLabelType.none,
+                        label: Text('None'),
+                      ),
+                      ButtonSegment<NavigationRailLabelType>(
+                        value: NavigationRailLabelType.selected,
+                        label: Text('Selected'),
+                      ),
+                      ButtonSegment<NavigationRailLabelType>(
+                        value: NavigationRailLabelType.all,
+                        label: Text('All'),
+                      ),
+                    ],
+                    selected: <NavigationRailLabelType>{
+                      labelType
+                    },
+                    onSelectionChanged:
+                        (Set<NavigationRailLabelType> newSelection) {
+                      setState(() {
+                        labelType = newSelection.first;
+                      });
+                    }),
```

Comment:
```suggestion
                SegmentedButton<NavigationRailLabelType>(
                  segments: const <ButtonSegment<NavigationRailLabelType>>[
                    ButtonSegment<NavigationRailLabelType>(
                      value: NavigationRailLabelType.none,
                      label: Text('None'),
                    ),
                    ButtonSegment<NavigationRailLabelType>(
                      value: NavigationRailLabelType.selected,
                      label: Text('Selected'),
                    ),
                    ButtonSegment<NavigationRailLabelType>(
                      value: NavigationRailLabelType.all,
                      label: Text('All'),
                    ),
                  ],
                  selected: <NavigationRailLabelType>{labelType},
                  onSelectionChanged:
                      (Set<NavigationRailLabelType> newSelection) {
                    setState(() {
                      labelType = newSelection.first;
                    });
                  },
                ),
```

This fixes indents and adds comma to the line.